cpp/src/hash/murmur_hash.cu

```diff
@@ -29,26 +29,29 @@ namespace cudf {
 namespace detail {
 
 std::unique_ptr<column> murmur_hash3_32(table_view const& input,
+                                        uint32_t seed,
                                         rmm::cuda_stream_view stream,
                                         rmm::mr::device_memory_resource* mr)
 {
   // TODO this should be UINT32
```

Comment:
```suggestion
```

Can this be removed now?

Reply:
Related discussion above: https://github.com/rapidsai/cudf/pull/10641#discussion_r849927422

Reply:
OK. I'm closing this to avoid duplicate tracking.